modules/ROOT/pages/rest-api-access-rbac-roles.adoc

```diff
@@ -20,321 +20,317 @@ include::partial$_show_page_header_block.adoc[]
 == Admin
 
 
-The table below identifies the RBAC roles required by Sync Gateway to execute endpoint actions.
-*Couchbase Server 7.0.2 Developer Preview* introduces new roles offering fine grained access control; existing roles will remain available.
+This table identifies the RBAC roles required by Sync Gateway to execute endpoint actions.
+*Couchbase Server 7.0.2 Developer Preview* introduces new roles offering fine-grained access control; existing roles will remain available.
 
 For more on RBAC roles, see the Couchbase Server documentation: {server-security-roles--xref}
 
 
 .REST API Admin Endpoints
-[#tbl-endpoint-role-crossref, cols="1,1,1,2,2", options="header"]
+[#tbl-endpoint-role-crossref, cols="1,1,2,2", options="header"]
 |===
 
-3+^| Rest API Endpoint
+2+^| Rest API Endpoint
 2+^| Couchbase Server Roles
-
-.>h| Scope
-.>h| Endpoint
-.>h| Method
-.>h| All  Versions
-h| 7.0.2 +
+| Endpoint | Method | All Versions | 7.0.2 +
```

Comment:
you've removed the `h|` which appears to be heading formatting https://docs.asciidoctor.org/asciidoc/latest/tables/format-cell-content/
is that no longer needed?

Reply:
oh I see, `%header` does this?